docs/release_notes/release_1_1_0.rst

```diff
@@ -0,0 +1,12 @@
+Release 1.1.0
```

Comment:
For avoiding sphinx warnings when not including this document anywhere, we added `:orphan:` at the beginning. To make sure that nobody mistakes this as a secret release they haven't seen, I usually also added something like "This release in currently in progress" or something along those lines.